packages/lime-docs/files/lime-example

```diff
@@ -38,66 +36,81 @@ config lime network
 #	option autoap_enabled 0				# Requires lime-ap-watchping installed. If enabled AP SSID is changed to ERROR when network issues
 #	option autoap_hosts "8.8.8.8 141.1.1.1"		# Requires lime-ap-watchping installed. Hosts used to check if the network is working fine
+	option use_odhcpd false
 
 ### WiFi general options
 
 config lime wifi
-#	option country 'ES'
-	option channel_2ghz '11'                    # May be either a list or a single option, in case of a list channel will be selected according to radio index
-	option channel_5ghz '48'			# Check for allowed channels on https://en.wikipedia.org/wiki/List_of_WLAN_channels#regulatory_tables5.0ghz
+#	option country 'ES'				# set this to your location countrycode, for example in Spain, setting ES allows you to use channel 13
+	option channel_2ghz '11'			# May be either a list or a single option, in case of a list channel will be selected according to radio index
+#	option channel_5ghz '48'			# Check for allowed channels on https://en.wikipedia.org/wiki/List_of_WLAN_channels#regulatory_tables5.0ghz
+	list channel_5ghz '157'				# If channel is a list, each entry of the list is used for a different radio
+	list channel_5ghz '48'
 #	option htmode_2ghz 'HT20'
 	option htmode_5ghz 'HT40'
-	option distance_2ghz '100'			# Distance between the ap and the furthest client in meters
-	option distance_5ghz '1000'			# Distance between this node and the furthest connected node in meters
-	list modes 'ap'						# You can specify if the mode is applied to all radios or just some particular type, for example apname_2ghz and adhoc_5ghz to have an access point in 2ghz and the adhoc in 5ghz. These suffixes (_2ghz and _5ghz) work for all modes
+	option distance_2ghz '100'			# Distance between the ap and the furthest client in meters, heavily affects performances
+	option distance_5ghz '1000'			# Distance between this node and the furthest connected node in meters, heavily affects performances
+	list modes 'ap'					# You can specify if the mode is applied to all radios or just some particular type, for example apname_2ghz and adhoc_5ghz to have an access point in 2ghz and the adhoc in 5ghz. These suffixes (_2ghz and _5ghz) work for all modes
 	list modes 'apname'
 	list modes 'ieee80211s'
 #	list modes 'adhoc'				# See below for adhoc configuration
 #	list modes 'client'				# See below for client configuration
-	option ap_ssid 'LibreMesh.org'
+	option ap_ssid 'LibreMesh.org'			# set here your network name, **this value is required even if AP is not used**, as it is used for calculating fields with %N1, %N2, %N3
+
 #	option ap_key 'SomeWPA2PskKey'
 #	option ap_encryption 'psk2'
-	option apname_ssid 'LibreMesh.org/%H'
+ 	option apname_ssid 'LibreMesh.org/%H'		# a user can connect to the named AP for avoiding roaming
```

Comment:
it probably sound better in English this way "to avoid roaming"

Reply:
done, thanks!